.github/workflows/test.yml

```diff
@@ -27,100 +27,120 @@ jobs:
   api-breakage:
     if: ${{ !(github.event.pull_request.draft || false) }}
     runs-on: ubuntu-latest
-    container: swift:5.8-jammy
+    container: swift:jammy
     steps:
       - name: Check out code
         uses: actions/checkout@v3
         with: { 'fetch-depth': 0 }
       - name: Run API breakage check action
-        uses: vapor/ci/.github/actions/ci-swift-check-api-breakage@reusable-workflows
+        uses: vapor/ci/.github/actions/ci-swift-check-api-breakage@main
+
+  code-coverage:
```

Comment:
This one is just for my curiosity as you probably already considered it, but why not use the reusable CI to get rid of separate `code-coverage` and `gh-codeql` jobs?

Reply:
Same reason as the other non-SQLite Fluent repos don't - the reusable workflow doesn't yet support the services specifications these repos need and I haven't yet made the reusable workflow capable of invoking "only" the CodeQL/coverage/API breakage jobs without any tests.